monai/data/augments/augments.py

```diff
@@ -9,28 +8,27 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
-
 """
 This contains the definitions of the commonly used argumentation functions. These apply operations to single instances
-of data objects, which are tuples of numpy arrays where the first dimension if the channel dimension and others are 
+of data objects, which are tuples of numpy arrays where the first dimension if the channel dimension and others are
 component, height/width (CHW), or height/width/depth (CHWD).
 """
 from functools import partial
+
 import numpy as np
-import scipy.ndimage
 import scipy.fftpack as ft
+import scipy.ndimage
 
-from monai.data.augments.decorators import augment, checkSegmentMargin
-from monai.utils.arrayutils import randChoice, rescaleArray, copypasteArrays, resizeCenter
-from monai.utils.convutils import oneHot
+from monai.data.augments.decorators import augment, check_segment_margin
+from monai.utils.arrayutils import (copypaste_arrays, rand_choice, rescale_array, resize_center)
+from monai.utils.convutils import one_hot
 
 try:
     from PIL import Image
 
-    pilAvailable = True
+    PILAvailable = True
 except ImportError:
-    pilAvailable = False
+    PILAvailable = False
```

Comment:
The problem with snake case is that you can't present acronyms correctly. I guess this should be pil_available but if you don't know that pil is an acronym it could confuse. Maybe change it to that for now?